README.md

```diff
@@ -208,6 +208,11 @@ On the backend, we use a Python package to extract the DB information from a DAT
     APP_BASE=backend
     SECRET_KEY=<SECRET_KEY>
 
+### Using an .env file to set environment variables
+
+REACT_APP_STRAPI_HOST=http://localhost:1337/
+REACT_APP_FRONTEND_HOST=http://localhost:3000/
+REACT_APP_STRAPI_TOKEN=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6MSwiaXNBZG1pbiI6dHJ1ZSwiaWF0IjoxNTgzNzk5NzU4LCJleHAiOjE1ODYzOTE3NTh9.dWR7T76GKYKaV5C6s8Yqma6mY6nsaklySF5PCDA4u7c
```

Comment:
We should likely not have the values of the keys here. 

something like this should do:
```
REACT_APP_STRAPI_HOST=
REACT_APP_FRONTEND_HOST=
REACT_APP_STRAPI_TOKEN=
```

also can you format this to be a code block by surrounding the three keys with three backticks ```